src/oss/confluent.sh

```diff
@@ -911,7 +911,26 @@ list_command() {
     fi
 }
 
+validate_java_version() {
+    local target_service=${1}
+    if [[ $target_service = "zookeeper" || target_service = "kafka" ]]; then
+        return
+    fi
+    local java_version=$(java -version 2>&1 | sed -n ';s/.* version "\(.*\)\.\(.*\)\..*"/\1\2/p;')
+    # Warn users if it is not Java7 or Java8, or if we can't figure it out.
+    if [[ $java_version -ne 17 && $java_version -ne 18 ]]; then
+        echo "WARNING: Java version 1.8 is recommended:"
+        echo "  See https://docs.confluent.io/current/installation/versions-interoperability.html for detailed version and interoperability information"
+    fi
+    # Fail fast if the version is definitely Java9
+    if [[ $java_version = 19 || $java_version = 110]]; then
+        echo "Java 1.9 is unsupported at this time: Confluent CLI will exit."
```

Comment:
Before exit, can we add the same doc link as above?

Reply:
It will execute both conditions, so it does the first block (prints link) and second block (exits)

Reply:
Good point, I missed that.